frontend-cert/js-projects/student-grade-tracker/script.js

```diff
@@ -0,0 +1,107 @@
+// start with an empty array - objects will be pushed into it later
+const students = [];
+
+// Have campers create 4 or more objects – but working with more than 3 might make thigns too repetitious and redundant
+const student1 = {
+  name: 'John',
+  scores: [88, 90, 85],
+  averageScore: null,
+  grade: '',
+};
+
+const student2 = {
+  name: 'Jane',
+  scores: [79, 88, 90],
+  averageScore: null,
+  grade: '',
+};
+
+const student3 = {
+  name: 'Zoe',
+  scores: [95, 92, 98],
+  averageScore: null,
+  grade: '',
+};
+
+const student4 = {
+  name: 'Vivian',
+  averageScore: null,
+  grade: '',
+};
+
+// introduce push here
+students.push(student1, student2, student3, student4);
+
+console.log(students);
+
+// introduce pop
+students.pop(); // remove the last student so three are two left to work with
+
+let student1TotalScore =
+  student1.scores[0] + student1.scores[1] + student1.scores[2];
+
+console.log(student1TotalScore);
+// repeat for the two other students
+
+let student1AverageScore = student1TotalScore / student1.scores.length;
+console.log(student1AverageScore);
+
+// round the average scpre to two decmal places
+let student1RoundedAvg = Math.floor(student1AverageScore * 100) / 100;
+console.log(student1RoundedAvg);
```

Comment:
In the numbers and math module, they will have learned about the `toFixed` method in the lectures. It looks like I left that out of the issue. 

But we should leave a note to allow for multiple correct answers when broken up into steps and tests.
Also, most campers will probably use this answer instead because they will have learned about it earlier.

```js
let student1RoundedAvg = (student1TotalScore / student1.scores.length).toFixed(2);
```
so we might consider using that in the final seed code instead